test/utils/query.js

```diff
@@ -0,0 +1,180 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import query from '../../src/utils/query';
+
+describe('query utils', () => {
+	const defaultEndpoint = 'accounts';
+	const defaultParameters = {
+		address: 'address1',
+		limit: 1,
+	};
+	const defaultArrayParameters = [
+		{
+			address: 'address1',
+			limit: 1,
+		},
+		{
+			address: 'address2',
+			limit: 1,
+		},
+	];
+
+	let apiClient;
+	let response;
+	describe('when the response does not have data', () => {
+		let queryResult;
```

Comment:
Since `queryResult` is used by almost all the suites, why not declare it in the outer scope along with the other `let`s?